### PR TITLE
feat: Use new buttons in guest links creation and add participants panels

### PR DESCRIPTION
**Pull request by thisisamir98**
image
image

### CODE REVIEW

File: src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx [hunk 1]
```diff
@@ -308,15 +310,16 @@ const AddParticipants: FC<AddParticipantsProps> = ({
...
           <div className="add-participants__footer">
-            <button
+            <Button
               type="button"
-              className="button button-full"
               disabled={!enabledAddAction}
               onClick={onAddParticipants}
               data-uie-name="do-create"
+              variant={ButtonVariant.PRIMARY}
+              role="button"
```
Comment (atomrc) [inline src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx]: Do we really need the `role=button` now that it's a proper button?

**Review by atomrc — APPROVED** [review]

File: src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx [hunk 2]
```diff
@@ -308,15 +310,16 @@ const AddParticipants: FC<AddParticipantsProps> = ({
...
         {isAddPeopleState && (
           <div className="add-participants__footer">
-            <button
+            <Button
               type="button"
-              className="button button-full"
               disabled={!enabledAddAction}
               onClick={onAddParticipants}
               data-uie-name="do-create"
+              variant={ButtonVariant.PRIMARY}
```
Comment (przemvs) [inline src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx]: You don't need pass variant primary, it's default variant for primary buttons :)

File: src/script/page/RightSidebar/GuestServicesOptions/components/GuestOptions/GuestOptions.tsx [hunk 3]
```diff
@@ -229,20 +231,19 @@ const GuestOptions: FC<GuestOptionsProps> = ({
...
-                    <Icon.Link />
-                  </span>
-
-                  <span className="panel__action-item__text" data-uie-name="do-create-link">
+                <div className="guest-options__content">
+                  <Button
+                    disabled={isRequestOngoing}
+                    variant={ButtonVariant.TERTIARY}
+                    type="button"
+                    role="button"
```
Comment (przemvs) [inline src/script/page/RightSidebar/GuestServicesOptions/components/GuestOptions/GuestOptions.tsx]: Role is necessary there, because it's native button :)

**Review by V-Gira — APPROVED** [review]
Ui-kit working as intended! Go ui-kit :rocket:

**Pull request merged**